File: 7-kyu/interactive-dictionary/python/solution.py
class Dictionary():
    def __init__(self):
        self._dict = {}
        
    def newentry(self, word, definition):
        self._dict[word] = definition
        
    def look(self, key):
        error_message = f"Can\'t find entry for {key}"
        return self._dict.get(key, error_message)

File: 7-kyu/interactive-dictionary/javascript/solution.js
class Dictionary {
  constructor() {
    this._dict = {};
  }
  
  newEntry(key, value) {
    this._dict[key] = value;
  }
  
  look(key) {
    if (this._dict[key] !== undefined) {
      return this._dict[key];
    } else {
      return `Can't find entry for ${key}`;
    }
  }
}